liblangutil/ErrorReporter.h

```diff
@@ -87,6 +87,34 @@ class ErrorReporter
 
 	void typeError(SourceLocation const& _location, std::string const& _description);
 
+	template <typename... Strings>
+	void typeErrors(SourceLocation const& _location, Strings const&... _descriptions)
+	{
+		std::initializer_list<std::string> const _descs = {_descriptions...};
+		std::string errorStr;
+
+		if (_descs.size() == 0)
+			return;
+
+		auto const ItFirst = _descs.begin();
```

Comment:
Style: local variables in camelCase with lowercase first letter.

Reply:
Even for constants?

Reply:
Being constant should not be something too special, I think. Classes and types are uppercase.